.github/workflows/unittests.yml

```
@@ -39,7 +39,7 @@
        - shell: bash -l {0}
          run: py.test -v libpysal --cov=libpysal --cov-report=xml
        - name: codecov (${{ matrix.os }}, ${{ matrix.environment-file }})
-         uses: codecov/codecov-action@v1.0.14
+         uses: codecov/codecov-action@v1.0.15
```

```suggestion
         uses: codecov/codecov-action@v1
```

@jGaboardi  If we use `codecov-action` like this, it will always pick the latest patch release. No need to bump version every now and then.

---

@martinfleis ah, thanks! I was wondering what the difference was with those. Is there any drawbacks for keeping it the way it is (other than manually bumping the versions once in a while)?

---

No, just unnecessary maintenance.